tests/rptest/services/openmessaging_benchmark.py

```diff
@@ -0,0 +1,195 @@
+        self.configuration = {
+            "name": "Ducktape workload",
+            "topics": 1,
+            "partitionsPerTopic": 3,
+            "messageSize": 1024,
+            "payloadFile": "payload/payload-1Kb.data",
+            "subscriptionsPerTopic": 1,
+            "producersPerTopic": 1,
+            "consumerPerSubscription": 1,
+            "producerRate": 0,
+            "consumerBacklogSizeGB": 0,
+            "testDurationMinutes": 15
+        }
+
+    def set_configuration(self, config):
+        for key, value in config.items():
+            self.configuration[key] = value
+
+    def _create_benchmark_workload_file(self, node):
+        workload_configuration = []
+        for key, value in self.configuration.items():
+            workload_configuration.append(f"{key}: {value}")
+        workload_file_content = "\n".join(workload_configuration)
+
+        self.logger.info(
+            "Open Messaging Bechmark: Workload configuration: \n %s",
+            workload_file_content)
+        node.account.create_file(
+            os.path.join(OpenMessagingBenchmark.OPENMESSAGING_DIR,
+                         OpenMessagingBenchmark.WORKLOAD_FILE),
+            workload_file_content)
+
+    def _set_benchmark_main_node(self, node):
+        main_node = self.redpanda.nodes[0].account.hostname
+        driver_file = os.path.join(OpenMessagingBenchmark.OPENMESSAGING_DIR,
+                                   OpenMessagingBenchmark.DRIVER_FILE)
+        node.account.ssh(
+            f"sed -i 's/DOCKER_ENTRY_NODE/{main_node}/g' {driver_file}")
+
+    def start_node(self, node):
+        wait_until(lambda: self.redpanda.healthy(),
+                   timeout_sec=50,
+                   backoff_sec=7,
+                   err_msg="Redpanda nodes in unhealthy state")
+
+        idx = self.idx(node)
+        self.logger.info("Open Messaging Bechmark: benchmark node - %d on %s",
+                         idx, node.account.hostname)
+
+        self.clean_node(node)
+
+        self._create_benchmark_workload_file(node)
+        self._set_benchmark_main_node(node)
+
+        worker_nodes = ""
+        for worker_node in self.workers.nodes:
+            worker_nodes += f"http://{worker_node.account.hostname}:{OpenMessagingBenchmarkWorker.PORT},"
+
+        self.logger.info(
+            f"Starting Open Messaging Bechmark with workers: {worker_nodes}")
+
+        start_cmd = f"cd {OpenMessagingBenchmark.OPENMESSAGING_DIR}; \
+                    bin/benchmark \
+                    --drivers {OpenMessagingBenchmark.DRIVER_FILE} \
+                    --workers {worker_nodes} \
+                    --output {OpenMessagingBenchmark.OUTPUT_FILE} \
+                    {OpenMessagingBenchmark.WORKLOAD_FILE} >> {OpenMessagingBenchmark.STDOUT_STDERR_CAPTURE} 2>&1 \
+                    & disown"
+
+        node.account.ssh(f"mkdir -p {OpenMessagingBenchmark.PERSISTENT_ROOT}")
+
+        node.account.ssh(start_cmd)
```

Comment:
it can be useful and make the service easier to use by waiting until it appears that the benchmark is fully started. here is an example:

https://github.com/redpanda-data/redpanda/blob/982a4d59a0fea14c5cfaec5206ce53ac9847a520/tests/rptest/services/redpanda.py#L167-L175